linker/shared/src/main/scala/org/scalajs/linker/backend/wasmemitter/LoaderContent.scala

```diff
@@ -88,6 +88,9 @@ const scalaJSHelpers = {
   undef: void 0,
   isUndef: (x) => x === (void 0),
 
+  // fileLevelThis (always undefined within ESModule)
+  fileLevelThis: this,
```

Comment:
Since we *only* support `ESModule`, have you considered directly *generating* an `undef` in `genJSGlobalRef` instead? That would simplify a few more things.

Reply:
Right, now I removed `fileLevelThis` from wasm backend, and `genJSGlobalRef` directly generate `undef`.